### PR TITLE
Bazelize some of the python testing helpers

### PR DESCRIPTION
**Pull request by fishcakez**
To improve testing of python grpc implementations when bring in this workspace as an external repostiory add some py_library rules.

@gnossen

### CODE REVIEW

**Comment by gnossen** [comment]
Thanks for the PR! First time I've seen the word "bazelize".

**Comment by gnossen** [comment]
[Known unrelated failure](https://github.com/grpc/grpc/issues/26322)

**Review by gnossen — APPROVED** [review]

**Comment by gnossen** [comment]
@lidizheng Could you please give this a second pair of eyes?

File: src/python/grpcio_testing/grpc_testing/BUILD.bazel
Comment (lidizheng) [inline src/python/grpcio_testing/grpc_testing/BUILD.bazel:1]: nit: We might need to include Copyright here, otherwise it will fail on import to Google. Same for other files.

File: src/python/grpcio_testing/grpc_testing/BUILD.bazel
Comment (lidizheng) [inline src/python/grpcio_testing/grpc_testing/BUILD.bazel:11]: optional: `:time` and `:common` can be private to this package?

File: src/python/grpcio_testing/grpc_testing/BUILD.bazel
Comment (lidizheng) [inline src/python/grpcio_testing/grpc_testing/BUILD.bazel:15]: I know creating a library for each file is more ideal for Bazel. But here we could use a glob to reduce future maintenance need. WDYT?

CC @gnossen

**Review by lidizheng — COMMENTED** [review]
Thanks for doing this!

File: src/python/grpcio_testing/grpc_testing/BUILD.bazel
Comment (gnossen) [reply src/python/grpcio_testing/grpc_testing/BUILD.bazel:1]: Good point. I'll look into why the copyright check didn't catch this.

File: src/python/grpcio_testing/grpc_testing/BUILD.bazel
Comment (gnossen) [reply src/python/grpcio_testing/grpc_testing/BUILD.bazel:15]: Considering that this directory has not changed in years and we have no plans to change it in the future, I think this is okay for the moment.

File: src/python/grpcio_testing/grpc_testing/BUILD.bazel
Comment (gnossen) [reply src/python/grpcio_testing/grpc_testing/BUILD.bazel:1]: I've put together https://github.com/grpc/grpc/pull/26353 to take care of this problem in general.

Comment (lidizheng) [reply src/python/grpcio_testing/grpc_testing/BUILD.bazel:1]: We might need to sync with master after #26353 is merged.

**Comment by rgs1** [comment]
We probably want to reopen...

**Pull request closed without merging**
